### ISSUE TITLE
Spike: post-EZID DOIs in Dataverse

### ISSUE THREAD

**Issue by johnhuck** [ualbertalib/metadata#165]
Investigate the state of response from Dataverse to the exit of EZID from the role of DOI registration. Is there code change in an upcoming release? What does UAL need to do to prepare? 

Situation awareness on this issue will help inform conversations with DataCite Canada in the context of Jupiter development.

**Comment by johnhuck** [comment]
According to the Dataverse installation guide for the [current version](http://guides.dataverse.org/en/latest/installation/config.html#persistent-identifiers-and-publishing-datasets) as well as for our [current version (4.5)](http://guides.dataverse.org/en/4.5/installation/config.html#persistent-identifiers-and-publishing-datasets), DOI provider is configurable. We should be able to change it when we need to, therefore.

**Comment by pdurbin** [comment]
I don't believe there's an issue at https://github.com/IQSS/dataverse/issues any work to be done in the Dataverse code or documentation regarding the exit of EZID, as you put it, but you are welcome to create one. The most recent conversation I'm aware of was a community call on 2017-09-12. Here are the notes: https://groups.google.com/d/msg/dataverse-community/Njy9b-jVJ9Q/Sy4IAY7-BwAJ . There was a thread about EZID before this call at https://groups.google.com/d/msg/dataverse-community/YB9NfL7S6t8/iXp73XbRAQAJ

**Comment by johnhuck** [comment]
Hi @pdurbin thanks for the extra links! I had seen the issue come up in the community Google group some time ago. We're just getting organized to transition from EZID to DataCite for our main (non-Dataverse) repository and so are also looking at what this will mean for our Dataverse instance.

Is there any work that needs to be done on the Dataverse code? It looked to me like it is just a re-configuration question, notwithstanding the three EZID features DataCite lacks that  Sherry Lake identified. Thanks for the advice!

**Comment by pdurbin** [comment]
@johnhuck to be honest, I'm not sure. I'm not very familiar with this part of the Dataverse code. If you want to create a GitHub issue in the Dataverse repo for this. please be my guest. I'm not sure what will have to change for production installations like yours but off the top of my head:

- Dataverse developers will need to switch their dev environments from EZID to something (presumably DataCite rather than Handle).
- All instances of "EZID" should be removed from the guide.
- The installer should no longer default to EZID.

These are all changes that would be part of the Dataverse repo. Maybe other changes will be required too. Not sure. Has anyone made the switch yet?
